packages/react-dom/src/__tests__/ReactTestUtilsAct-test.js

```diff
@@ -0,0 +1,326 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ */
+
+let React;
+let ReactDOM;
+let ReactTestUtils;
+let act;
+
+jest.useRealTimers();
```

Comment:
I don't get it. Why can't you use jest's fake timers?

Reply:
OSS can’t/don’t use fakeTimers reliably, so I want to run tests in an environment as close to theirs as possible. (Specifically for the async/await test, which started the need for this pr)

Reply:
Can you move this to the specific test(s) where you rely on native timers then?

Reply:
Missed erplying to this. It's all the tests, except the ones that use fake timers. I want to be explicit about using fake timers in this suite.

Reply:
It is very hard to see if you cover all case for both environments. By default I expect there to be one file for each environment and every test mirrored in each file so that I can see that you've covered all cases in each environment and how you expect any case to be ported.

If some test needs to test something specific that is only relevant in one environment sure, but I expect code to mostly work the same in either environment.

Therefore splitting these into two separate files with a clear name in the file name which environment you're testing would be good.

Reply:
That's fair, I could do that.